src/ProxyManager/ProxyGenerator/LazyLoadingGhostGenerator.php

```diff
@@ -60,7 +60,7 @@ class LazyLoadingGhostGenerator implements ProxyGeneratorInterface
     /**
      * {@inheritDoc}
      */
-    public function generate(ReflectionClass $originalClass, ClassGenerator $classGenerator)
+    public function generate(ReflectionClass $originalClass, ClassGenerator $classGenerator, array $properties = [])
```

Comment:
I think it should be `array $options`, and a key in it would be `skippedProperties` or such